content/docs/tutorials/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Tutorials"
+linkTitle: "tutorials"
```

Comment:
The linkTitle is used in the sidebar, which means that currently all of our sidebar navigation is lowercase.. can you go through and update these?

Reply:
:cry: